### DISCUSSION TITLE
How can you guys send solutions with only 1 LOC?

### DISCUSSION THREAD

**Discussion by Folskyy** [codewars/codewars.com#3184] [Q&A]
i see people sending solutions in python and C++ that  only have the return line. How do you guys does that codes? You already know the functions/methods to use or search for new ones?

**Comment by hobovsky** [comment]
Please mind that Codewars Github is not meant for help with general programming or learning questions, or for help with solving kata. It's meant to discuss matters related to Codewars platform, the website, its features, etc.

For help with this type of questions as in your post, please visit your favorite general programming forum, a code-golfing forum, or you can ask it in the `#beginner` section of Codewars Discord.

I am going to close this discussion as off-topic for the Codewars Github.
